Source/monster.h

```diff
@@ -338,6 +338,12 @@ struct Monster { // note: missing field _mAFNum
 	bool isWalking() const;
 	bool isImmune(missile_id mitype) const;
 	bool isResistant(missile_id mitype) const;
+
+	/**
+	 * Is this a player's golem?
+	 */
+	[[nodiscard]] bool isPlayerOwned() const;

```

Comment:
Somehow I don't much like the name. For one thing it makes me think that it might be specific to `MyPlayer`. Maybe `isPlayerMinion` or `IsPlayerControlled` would be better.

Reply:
Haha, another case where I actually had it as `isPlayerControlled` then changed it because I was worried people might think it means a player is actually moving it around directly. `isPlayerMinion` sounds good.

Reply:
Golems actually controlled by the player as they will walk in what ever cardinal direction the player is looking :)

Reply:
I never realised that 🤯